reframe/REFRAME_KNOWN_METHODS.md

```diff
@@ -269,7 +269,7 @@ A message for indicating that the client is able to act as a provider for a give
 
 ```ipldsch
     type ProvideRequest struct 

```

Comment:
It doesn't seem needed at the moment, but flagging that down the line I could see you wanting to tag a provide request a particular way, or allow some data passed back in the response to allow bulk manipulation if we want to update or delete a provide message (e.g. as storetheindex allows).